paper/paper.md

```diff
@@ -86,18 +90,19 @@ bibliography: paper.bib
 
 
 # Statement of need
 
-The Molecular Simulation Design Framework (MoSDeF) is a suite of software tailored to facilitate the initialization of chemical and biomolecular systems for computational simulations [@cummings2021opena]. These tools were developed to specifically address a critical aspect of the (ir)reproducibility issue within the molecular simulation community — namely, the insufficient documentation of the structure preparation process and force field parameter implementation [@thompson2020towards]. The initialization step, often performed through Graphical User Interfaces (GUI) or via the use of ad-hoc, unpublished, and unreviewed code, poses the risk of introducing irreproducible and untraceable errors[@baker2016reproducibility]. By providing general-purposed and standardized tools that build and parameterize molecular systems for molecular simulations, directly support  various molecular dynamics (MD) and Monte Carlo (MC) engines, MoSDeF aims to trivialize the describing and disseminating such processes without creating extra burdens for computational simulation researchers [@cummings2021opena].
+The General Molecular Simulation Object (GMSO) is a component of the Molecular Simulation Design Framework (MoSDeF), provides a framework and utilities for storing, manipulating, and outputting of molecular systems. The Molecular Simulation Design Framework (MoSDeF) is a suite of software tailored to facilitate the initialization of chemical and biomolecular systems for computational simulations [@cummings2021opena]. These tools were developed to specifically address a critical aspect of the (ir)reproducibility issue within the molecular simulation community — namely, the insufficient documentation of the structure preparation process and force field parameter implementation [@thompson2020towards]. The initialization step, often performed through Graphical User Interfaces (GUI) or via the use of ad-hoc, unpublished, and unreviewed code, poses the risk of introducing irreproducible and untraceable errors[@baker2016reproducibility]. By providing general-purposed and standardized tools that build and parameterize molecular systems for molecular simulations, directly support  various molecular dynamics (MD) and Monte Carlo (MC) engines, MoSDeF aims to trivialize the describing and disseminating such processes without creating extra burdens for computational simulation researchers [@cummings2021opena].
```

Comment:
The changes look good!

Just a couple of things in this paragraph:

1. In back-to-back sentences we both spell out and show the abbreviation for MoSDeF. Is that by choice? We could just use the abbreviation the second time if that seems easier to read.

2. Also in this paragraph, we give abbreviations for molecular dynamics and monte carlo, but we never use them again anywhere else. We could drop the (MD) and (MC). That's one less thing for the reader to keep track of.

Reply:
good catch! those were not intentional. pushing a fix now

Reply:
done.